cmd/performance-profile-creator/cmd/root.go

```diff
@@ -94,7 +103,13 @@ func NewRootCommand() *cobra.Command {
 					return fmt.Errorf("failed to parse the cluster data: %v", err)
 				}
 
-				showClusterData(cluster)
+				clusterInfo := makeClusterInfoFromClusterData(cluster)
+				// intentionally undocumented and unsupported output to make e2e testing easier.
+				if infoMode == infoModeJSON {
+					showClusterInfoJSON(clusterInfo)
```

Comment:
This we miss the recommendation here: https://github.com/openshift-kni/performance-addon-operators/pull/628#discussion_r635967424? Not blocking in any way (can be done in another PR)?

Reply:
my fault I missed it. Fixing.

Reply:
Ignore this comment: this is my bad